docs/csharp/language-reference/compiler-options/how-to-set-environment-variables-for-the-visual-studio-command-line.md

```diff
@@ -40,20 +40,24 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # How to: Set Environment Variables for the Visual Studio Command Line
-The vsvars32.bat file sets the appropriate environment variables to enable command-line builds. For more information about vsvars32.bat, see [Knowledge Base article Q248802](http://go.microsoft.com/fwlink/?LinkId=225042).  
+
+The VsDevCmd.bat file sets the appropriate environment variables to enable command-line builds. For more information about VsDevCmd.bat, see [Knowledge Base article Q248802](http://go.microsoft.com/fwlink/?LinkId=225042).  
+
+> [!NOTE]
+> The VsDevCmd.bat file is a new file delivered with Visual Studio 2017. Visual Studio 2015 and earlier versions used VSVARS32.bat for the same purpose. This file was stored in Program Files\Microsoft Visual Studio\*Version*\Common7\Tools or Program Files (x86)\Microsoft Visual Studio\*Version*\Common7\Tools.
   
- If the current version of Visual Studio is installed on a computer that also has an earlier version of Visual Studio, you should not run vsvars32.bat or vcvars32.bat from different versions in the same Command Prompt window.  
+If the current version of Visual Studio is installed on a computer that also has an earlier version of Visual Studio, you should not run VsDevCmd.bat and VSVARS32.BAT from different versions in the same Command Prompt window.
   
-### To run VSVARS32.BAT  
+### To run VsDevCmd.BAT  
   
-1.  From the **Start** menu, open the **Developer Command Prompt for VS2012**.  
+1.  From the **Start** menu, open the **Developer Command Prompt for VS2017**.  It's in the **Visual Studio 2017** folder
   
-2.  Change to the Program Files\Microsoft Visual Studio *Version*\Common7\Tools or Program Files (x86)\Microsoft Visual Studio *Version*\Common7\Tools subdirectory of your installation.  
+2.  Change to the Program Files\Microsoft Visual Studio\*Version*\*Offering*\Common7\Tools or Program Files (x86)\Microsoft Visual Studio\*Version*\*Offering*\Common7\Tools subdirectory of your installation.  (*Version* is *2017* for the current version. *Offering* is one of *Enterprise*, *Professional* or *Community*.)
```

Comment:
weird styling here too
![image](https://user-images.githubusercontent.com/12971179/31039737-fc9c8ca8-a534-11e7-8990-ca205af87b2a.png)

@rpetrusha I think we usually call out versions in path in a different way no? like putting in tags or something.

Reply:
@BillWagner, @mairaw, I don't think we've established a way to do it here. Previously, we used two conventions. One was <replaceable> tag/character style, which italicized the string using the current font. We also used a notation like `\Program Files\Microsoft Visual Studio\<version>\...`. 

Here, it would be clear if this appears in italics, as you'd intended it to, @BillWagner . The question is how to do that. The initial issues in each case are caused by the initial `\*` which is rendering as `*`, so I think I'd try either replacing * and * with <i> and </i>, or escaping the backslash that precedes ("\\*") and follows the asterisk ("*\\") to see what happens.